### PR TITLE
updating package.json with latest stable API version for chalk

### PR DESCRIPTION
**Pull request by lirantal**
* updating package.json with latest stable API version for chalk
* forcing colorful formatting for chalk when app starts

commiting this due to a similar PR request on generator meanjs project.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -7,6 +7,9 @@ var init = require('./config/init')(),
 	mongoose = require('mongoose'),
 	chalk = require('chalk');
 
+// Enable, or force chalk to format colorful texts
+chalk.enabled = true;
```
Comment (ilanbiala) [inline server.js]: Shouldn't this be set in config/express.js?

**Comment by lirantal** [comment]
@ilanbiala it's better to put it in config/init.js

**Comment by ilanbiala** [comment]
Why force set it at all, it is automatically detected?

**Comment by lirantal** [comment]
In my console, it wasn't although it previously did detect it automatically, so I'm thinking maybe something changed with the detection settings in chalk.

**Comment by ilanbiala** [comment]
@lirantal what system? My Mac 10.9.5 works fine without it.

**Comment by lirantal** [comment]
I'm running an Ubuntu 14 with an XTerm. It did use to work before... I can't say for sure what changed.

**Comment by ilanbiala** [comment]
Can you look at [this issue](https://github.com/sindresorhus/chalk/issues/51) and see if you have a similar problem?

**Comment by lirantal** [comment]
Thanks, that sheds 'some' light, not really sure.
If I run 'grunt' I don't get the colors (which I assume thats what the issue you were referring to mentions with piping input to the script), but rather if I run 'node server.js' then it works.

Thing is, I do recall that I had colors when running just grunt, and for you it works if you run just grunt and remove the color forcing, right?

**Comment by ilanbiala** [comment]
Yeah it works. Have you tried on 0.4.0?

**Comment by lirantal** [comment]
No I didn't. What do you think, should we keep it without forcing the colors or should we?

**Comment by ilanbiala** [comment]
I think we shouldn't force the colors for everyone if most people's machines don't need it.

**Comment by lirantal** [comment]
@ilanbiala I had some issues squashing commits here due to already pushing this branch to github apparently. Anyway, I just submitted a new PR for the package.json change and will close this one.

new PR: https://github.com/meanjs/mean/pull/457

**Pull request closed without merging**